airflow/contrib/hooks/aws_glue_job_hook.py

```diff
@@ -136,56 +136,59 @@ def job_completion(self, job_name=None, run_id=None):
         :param run_id:
         :return:
         """
-        glue_client = self.get_conn()
-        job_status = glue_client.get_job_run(
-            JobName=job_name,
-            RunId=run_id,
-            PredecessorsIncluded=True
-        )
-        job_run_state = job_status['JobRun']['JobRunState']
-        failed = job_run_state == 'FAILED'
-        stopped = job_run_state == 'STOPPED'
-        completed = job_run_state == 'SUCCEEDED'
-
         while True:
+            glue_client = self.get_conn()
+            job_status = glue_client.get_job_run(
+                JobName=job_name,
+                RunId=run_id,
+                PredecessorsIncluded=True
+            )
+            job_run_state = job_status['JobRun']['JobRunState']
+            failed = job_run_state == 'FAILED'
+            stopped = job_run_state == 'STOPPED'
+            completed = job_run_state == 'SUCCEEDED'
             if failed or stopped or completed:
                 self.log.info("Exiting Job {} Run State: {}"
                               .format(run_id, job_run_state))
                 return {'JobRunState': job_run_state, 'JobRunId': run_id}
             else:
-                self.log.info("Polling for AWS Glue Job {} current run state"
-                              .format(job_name))
+                self.log.info("Polling for AWS Glue Job {} current run state with status {}"
```

Comment:
```suggestion
                self.log.info("Polling for AWS Glue Job %s current run state with status %s", job_name,job_run_state)

```
I think, you should avoid string formatting. 
Reference: https://github.com/apache/airflow/pull/4804

Reply:
@mik-laj can you please guideme,how can i submit my changes to airflow original repo. Auther of my fork isnt merging my pull request

Reply:
You must create a ticket in JIRA and change commit message. It should have following format: 
```
[AIRFLOW-1234] Commit message
````
In the next step, you can create a PR for Apache using the link below:
https://github.com/OElesin/incubator-airflow/compare/master...abdulbasitds:master